tests/test_pyrocky.py

```diff
@@ -20,35 +20,44 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 import os
-import sys
+import time
 
+from Pyro5.errors import CommunicationError
 import pytest
 
 import ansys.rocky.core as pyrocky
-from ansys.rocky.core.client import DEFAULT_SERVER_PORT
-from ansys.rocky.core.launcher import RockyLaunchError
+from ansys.rocky.core.client import RockyClient
+from ansys.rocky.core.exceptions import PyRockyError, RockyLaunchError
+from ansys.rocky.core.launcher import _CONNECT_TO_SERVER_TIMEOUT, _find_executable
 from ansys.rocky.core.rocky_api_proxies import ApiExportToolkitProxy
 
-if os.getenv("ANSYS_VERSION"):
-    VERSION = int(os.getenv("ANSYS_VERSION"))
-else:
-    raise EnvironmentError("No ANSYS_VERSION environment variable set.")
 
-FREEFLOW_VERSION = 251
+@pytest.fixture
+def app_session():
+    """Launch and close Rocky/FreeFlow session(s)."""
+    clients: list[RockyClient] = []
```

Comment:
Add a comment explaining that his must support multiple clients being launched concurrently, that's why we need this list of clients.